core/embed/extmod/modtrezorui/display.c

```diff
@@ -349,6 +349,36 @@ void display_icon(int x, int y, int w, int h, const void *data,
   }
 }
 
+bool display_toif_info(const uint8_t *data, uint32_t len, uint16_t *out_w,
+                       uint16_t *out_h, bool *out_grayscale) {
+  if (len < 8 || memcmp(data, "TOI", 3) != 0) {
```

Comment:
I wonder if we should increase 8 to 12 here.

Reply:
Since `datasize` size mandatory I think yes.

Reply:
Btw maybe we could add a comment and link to `docs/misc/toif.md`?

Reply:
Fixed 277cb136.